code/game/machinery/cell_charger.dm

```diff
@@ -91,7 +91,7 @@
 		if(!charging || (stat & (BROKEN|NOPOWER)) || !anchored)
 			return
 
-		use_power(2000)		//this used to use CELLRATE, but CELLRATE is fucking awful. feel free to fix this properly!
-		charging.give(1750)	//inefficiency.
+		use_power(200)		//this used to use CELLRATE, but CELLRATE is fucking awful. feel free to fix this properly!
+		charging.give(175)	//inefficiency.

```

Comment:
Making power generation complicated isn't really necessary and is kinda overtly silly when powernets are magical things to begin with.

If this is ever at a level that it has meaningful impact, like what you're mentioning, then it's going to totally drain an APC any time you use it, which is patently ridiculous.